src/ArrayInterface.jl

```diff
@@ -31,6 +31,12 @@ import ArrayInterfaceCore: ismutable, can_change_size, can_setindex, deleteat, i
 # constants
 import ArrayInterfaceCore: MatAdjTrans, VecAdjTrans, UpTri, LoTri
 
+using ArrayInterfaceBandedMatrices
+using ArrayInterfaceBlockBandedMatrices
+using ArrayInterfaceCUDA
+using ArrayInterfaceOffsetArrays
+using ArrayInterfaceStaticArrays
+using ArrayInterfaceTracker
```

Comment:
No, those were removed on purpose. It would be really breaking to the ecosystem if ArrayInterface just started requiring CUDA and all of that. Instead, ArrayInterface should be what we're calling ArrayInterfaceStatic, allowing the core to just be the minimal definitions and quick methods that have no loading impact (i.e., all of the non-static stuff).